zjit/src/hir.rs

```diff
@@ -1020,6 +1020,30 @@ impl Insn {
 pub struct InsnPrinter<'a> {
     inner: Insn,
     ptr_map: &'a PtrPrintMap,
+    iseq: Option<IseqPtr>,
+}
+
+/// Get the name of a local variable given iseq, level, and ep_offset.
+/// Returns
+/// - `:name, ` if iseq is available and name is a real identifier,
+/// - `<anon>, ` for anonymous locals.
+/// - empty string if iseq is not available.
+///
+/// This mimics local_var_name() from iseq.c.
+fn get_local_var_name_for_printer(iseq: Option<IseqPtr>, level: u32, ep_offset: u32) -> String {
```

Comment:
This can return `Option<String>` then callers can do `name.map(|x| format!("{x}, ")).unwrap_or_else("")` instead so we can actually get local names separately from the comma

Reply:
oh `map_or`